pallets/providers/src/mock.rs

```diff
@@ -69,7 +69,7 @@ impl pallet_balances::Config for Test {
     type RuntimeHoldReason = RuntimeHoldReason;
     type RuntimeFreezeReason = ();
     type FreezeIdentifier = ();
-    type MaxHolds = ConstU32<10>;
+    type MaxHolds = ConstU32<100>;
```

Comment:
I can't recall precisely, but the substrate runtime was panicking due to this number being too low since the providers pallet introduced new hold reasons.

Reply:
Ok cool